src/reducer/videoReducer.jsx

```diff
@@ -0,0 +1,30 @@
+const VideoReducer = (state, action) => {
+    switch (action.type) {
+        case "LOAD_VIDEO":
+            return { ...state, videos: action.payload };
+        case 'LIKE':{
+            return {
+                ...state, liked: [...state.liked, {...action.payload}]
+            }
+        }
+        case 'WATCH_LATER':{
+            return {
+                ...state, watchlater: [...state.watchlater, {...action.payload}]
+            }
+        }
+        case 'ADD_TO_PLAYLIST':{
+            return {
+                ...state, playlist: [...state.playlist, {...action.payload}]
+            }
+        }
+        case 'HISTORY':{
+            return {
+                ...state, history: [...state.history, {...action.payload}]
+            }
+        }
+        default:
+            return state;
```

Comment:
Instead of returning state , you can also throw an error

Reply:
Will look into that.